tests/src/test/java/alluxio/client/fs/LocalFirstPolicyIntegrationTest.java

```diff
@@ -75,12 +75,14 @@ public void after() {
   @Test
   public void localNodeFirst() throws Exception {
     AlluxioMasterProcess master = AlluxioMasterProcess.Factory.create();
-    WorkerProcess worker1 = AlluxioWorkerProcess.Factory
-        .create(TieredIdentityFactory.fromString("node=node1,rack=rack1",
+    Whitebox.setInternalState(TieredIdentityFactory.class, "sInstance",
```

Comment:
Doesn't guice have better ways for testing instead of using whitebox?

whitebox for testing is easy to break and is not recommended

Reply:
It's not a good way to use WhiteBox, but I think it would be better to do a clean up for all the tests in the next PR.